samples/cpp/benchmark_app/utils.cpp

```diff
@@ -690,12 +690,32 @@ void dump_config(const std::string& filename, const std::map<std::string, ov::An
         throw std::runtime_error("Error: Can't open config file : " + filename);
     for (auto device_it = config.begin(); device_it != config.end(); ++device_it) {
         fs << plugin_to_opencv_format(device_it->first) << "{:";
+        std::map<std::string, ov::AnyMap> device_properties;
```

Comment:
This is not for this PR.
Is there a reason to have OpenCV's configs? There's always `nlohmann::json`, we could just always use this and remove OpenCV implementation.

Reply:
The handling logic for OpenCV's configs is come from master. I don't why use this OpenCV implementation here. I just update part of original code that needs to dump the ov::device::properties.

Reply:
We have replaced OpenCV by nlohmann_json in this PR https://github.com/openvinotoolkit/openvino/pull/9648

Reply:
@nkogteva Currently if there is OpenCV installation on the system, and benchmark_app will be build and go to this piece of code in runtime, shall we (1) remove this piece of code or  (2) fix it and remove it in next step?

Reply:
I saw the code `#ifdef USE_OPENCV` is still in utils.cpp. So should I just remove dump/load functions for OpenCV configs in this PR?

Reply:
> @nkogteva Currently if there is OpenCV installation on the system, and benchmark_app will be build and go to this piece of code in runtime, shall we (1) remove this piece of code or (2) fix it and remove it in next step?

According to the problem statement in original ticket:
```
Benchmark_app config files for devices are currently read and dumped via OpenCV which allows to do it in XML, JSON and YAML formats. Due to OpenCV was removed from default build of OpenVINO, it's necassary to reimplement config reader and dumper using new JSON library (which recently has been included in OpenVINO thirdparty - https://github.com/nlohmann/json).
Note that XML and YAML formats won't be supported by new implementation. At first time there will be possible to use old reader and dumper for backward-compatibiliy with warning that this way was deprecated and will be removed soon.
```

So we should remove OpenCV support for json parsing in separate PR. Also as far as I remember OpenCV parser has significant limitations. You can see warnings in code:
```
        if (str.find("_") != std::string::npos) {
            slog::warn
                << "Device name contains \"_\" and will be changed during loading of configuration due to limitations."
                   "This configuration file could not be loaded correctly."
                << slog::endl;
        }
```

Reply:
I will revert the changes within the dump/load function of OpenCV version in this PR.